addons/hr_payroll/models/hr_payroll.py

```diff
@@ -223,24 +224,25 @@ def _check_dates(self):
             raise ValidationError(_("Payslip 'Date From' must be before 'Date To'."))
 
     @api.multi
-    def cancel_sheet(self):
-        return self.write({'state': 'cancel'})
+    def action_payslip_draft(self):
+        return self.write({'state': 'draft'})
 
     @api.multi
-    def process_sheet(self):
-        return self.write({'paid': True, 'state': 'done'})
+    def action_payslip_done(self):
+        self.compute_sheet()
```

Comment:
def check_done(self):
 . return True;

I might have forgotten to remove the method